set.mm

```diff
@@ -529430,2958 +529515,2965 @@ standard proof of the implication in modal logic (B5 ` => ` 4).  Its dual
+      BGZQAHBIZSQACIZFZBGTRUBBAUAQACDJKLABCMNABCOP $.
   $}
 
   ${
-    $d ph x y $.
-    @( Proof of ~ ax-5 from ~ ax-bj-00 (note that the use of ~ ax-5 is via
-       ~ 19.8v , which is a form of ~ sp , and ~ ax-5 and the auxiliary axioms
-       are used via the axiom of twoness ~ dtru ).  (Contributed by BJ,
-       19-Mar-2021.)  (Proof modification is discouraged.)
-       (New usage is discouraged.) @)
-    bj-ax5 $p |- ( ph -> A. x ph ) $=
-      ( vy wex wal 19.8v weq wi wn wa ax6ev dtru pm3.2i annim mpbi ax-bj-00 syl
-      mtpor ) AABDZABEZABFBCGZBDZUABEZHZSTHUBUCIZJUDIUBUEBCKBCLMUBUCNOUAABPRQ
-      $.
+    $d x y $.
+    $( Version of ~ hbsb2 with a disjoint variable condition, which does not
+       require ~ ax-13 , and removal of ~ ax-13 from ~ hbs1 .  (Contributed by
+       BJ, 23-Jun-2019.)  (Proof modification is discouraged.) $)
+    bj-hbs1 $p |- ( [ y / x ] ph -> A. x [ y / x ] ph ) $=
+      ( wsb weq wi wal sb6 biimpri axc4i sylbi ) ABCDZBCEAFZBGZLBGABCHZMLBLNOIJ
+      K $.
   $}
-$)
 
   ${
-    $d x z $.  $d y z $.  $d ph z $.  $d ps z $.
-    bj-dvelimdv.nf $e |- ( ph -> F/ x ch ) $.
-    bj-dvelimdv.is $e |- ( z = y -> ( ch <-> ps ) ) $.
-    $( Deduction form of ~ dvelim with disjoint variable conditions.  Uncurried
-       (imported) form of ~ bj-dvelimdv1 .  Typically, ` z ` is a fresh
-       variable used for the implicit substitution hypothesis that results in
-       ` ch ` (namely, ` ps ` can be thought as ` ps ( x , y ) ` and ` ch ` as
-       ` ps ( x , z ) ` ).  So the theorem says that if x is effectively free
-       in ` ps ( x , z ) ` , then if x and y are not the same variable, then
-       ` x ` is also effectively free in ` ps ( x , y ) ` , in a context
-       ` ph ` .
-
-       One can weaken the implicit substitution hypothesis by adding the
-       antecedent ` ph ` but this typically does not make the theorem much more
-       useful.  Similarly, one could use nonfreeness hypotheses instead of
-       disjoint variable conditions but since this result is typically used
-       when ` z ` is a dummy variable, this would not be of much benefit.  One
-       could also remove DV ` ( x , z ) ` since in the proof ~ nfv can be
-       replaced with ~ nfal followed by ~ nfn .
+    $d x y $.
+    $( Version of ~ nfsb2 with a disjoint variable condition, which does not
+       require ~ ax-13 , and removal of ~ ax-13 from ~ nfs1v .  (Contributed by
+       BJ, 24-Jun-2019.)  (Proof modification is discouraged.) $)
+    bj-nfs1v $p |- F/ x [ y / x ] ph $=
+      ( wsb bj-hbs1 nf5i ) ABCDBABCEF $.
+  $}
 
-       Remark: ~ nfald uses ~ ax-11 ; it might be possible to inline and use
-       ~ ax11w instead, but there is still a use via ~ 19.12 anyway.
-       (Contributed by BJ, 20-Oct-2021.)
+  ${
+    $d x y $.
+    $( Version of ~ hbsb2a with a disjoint variable condition, which does not
+       require ~ ax-13 .  (Contributed by BJ, 11-Sep-2019.)
```

Comment:
Theorem [bj-hbsb2av](https://us.metamath.org/mpeuni/bj-hbsb2av.html) does depend on ax-13 because it uses [sb4a](https://us.metamath.org/mpeuni/sb4a.html), I don't know if there are similar versions without ax-13 that can be used here.

Reply:
The same `git checkout 561248a805a60cd9eedef4d31b6ccc1447b07798` shows that indeed `sb4a` didn't require ax-13, but now does because of the change of df-nf.
Maybe we should add
```

  ${  $d x t $.
    $( PLEASE PUT DESCRIPTION HERE. $)
    sb4av $p |- ( [ t / x ] A. t ph -> A. x ( x = t -> ph ) ) $=
    ( wal wsb weq wi sp sbimi sb6 sylib ) ACDZBCEABCEBCFAGBDLABCACHIABCJK $.
  $}
```
?

Reply:
Added sb4av in the PR: https://github.com/metamath/set.mm/pull/3697